component/card/card.css

```diff
@@ -0,0 +1,176 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap");
+@import url("../button/button.css");
+html,
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  overflow-x: hidden;
+}
+* {
+  scroll-behavior: smooth;
+}
+
+body {
+  font-family: "Montserrat", sans-serif;
+}
+
+.card h2,
+h4,
+h5 {
+  margin: 0;
+}
+.responsive-img {
+  max-width: 100%;
+}
+.card {
+  width: 17rem;
+  height: 100%;
+  box-sizing: border-box;
+  position: relative;
+  line-height: 24px;
+  cursor: pointer;
```

Comment:
You can add a box-shadow or a border for the card since the background-color of the card is getting mixed with the background-colour of the window.

Reply:
thanks for this